frontend/src/common/entities.ts

```diff
@@ -75,7 +77,7 @@ export interface Author {
 
 export interface Collection {
   access_type: ACCESS_TYPE;
-  consortia: string[];
+  consortia: CONSORTIA[];
```

Comment:
@tihuan I apologise - the changes made to PR https://github.com/chanzuckerberg/single-cell-data-portal/pull/3875 were accidentally wiped in a rebase. 😬